### ISSUE TITLE
t55xx chip support

### ISSUE THREAD

**Issue by GameTec-live** [RfidResearchGroup/ChameleonUltra#75]
The Chameleon Ultra should support detecting and configuring t55xx chips...

**Comment by GameTec-live** [comment]
> Already mentioned in https://github.com/RfidResearchGroup/ChameleonUltra/blob/main/Technical_White_Paper.md#low-frequency-simulation roadmap, no need to open an issue for each not-yet-implemented one ;)

true, but currently hf is getting all of the attention (which is fine?), but imo t55xx is a kinda basic lf feature?
good that its in the roadmap though

**Comment by doegox** [comment]
it's first on the list behind EM...
